docs/api.md

```diff
@@ -362,6 +362,7 @@ A valid request will return a JSON response with these properties:
 - `access_token`: A string that can be used for authorized requests to service providers.
 - `scope`: A string of space-separated permissions that this token has. May differ from requested scopes, since user can deny permissions.
 - `token_type`: A string representing the token type. Currently will always be "bearer".
+- `last_auth_at`: An integer giving the time at which the user last authenticated to the Firefox Accounts server, as **seconds since epoch**.
```

Comment:
is this a UTC unix timestamp? If not, can you make it one? It would be a lot easier to work with.